Live/LiveEventWithDVR/Program.cs

```diff
@@ -1,663 +1,562 @@
+    elapsedTime = String.Format(":{0:00}.{1:00}", watch.Elapsed.Seconds, watch.Elapsed.Milliseconds / 10);
+    Console.WriteLine($"Start Live Event run time : {elapsedTime}");
+    Console.WriteLine();
+
+    // Refresh the liveEvent object's settings after starting it...
+    liveEvent = await mediaServicesAccount.GetMediaLiveEventAsync(liveEventName);
+
+
+    #region GetIngestUrl
+    // Get the RTMP ingest URL to configure in OBS Studio. 
+    // The endpoints is a collection of RTMP primary and secondary, and RTMPS primary and secondary URLs. 
+    // to get the primary secure RTMPS, it is usually going to be index 3, but you could add a loop here to confirm...
+    string ingestUrl = liveEvent.Data.Input.Endpoints.First().Uri.ToString();
+    Console.WriteLine($"The RTMP ingest URL to enter into OBS Studio is:");
+    Console.WriteLine($"\t{ingestUrl}");
+    Console.WriteLine("Make sure to enter a Stream Key into the OBS studio settings. It can be any value or you can repeat the accessToken used in the ingest URL path.");
+    Console.WriteLine();
+    #endregion
+
+    #region GetPreviewURLs
+    // Use the previewEndpoint to preview and verify
+    // that the input from the encoder is actually being received
+    // The preview endpoint URL also support the addition of various format strings for HLS (format=m3u8-cmaf) and DASH (format=mpd-time-cmaf) for example.
+    // The default manifest is Smooth. 
+    string previewEndpoint = liveEvent.Data.Preview.Endpoints.First().Uri.ToString();
+    Console.WriteLine($"The preview url is:");
+    Console.WriteLine($"\t{previewEndpoint}");
+    Console.WriteLine();
+
+    Console.WriteLine($"Open the live preview in your browser and use the Azure Media Player to monitor the preview playback:");
+    Console.WriteLine($"\thttps://ampdemo.azureedge.net/?url={previewEndpoint}&heuristicprofile=lowlatency");
+    Console.WriteLine();
+    #endregion
+
+    Console.WriteLine("Start the live stream now, sending the input to the ingest url and verify that it is arriving with the preview url.");
+    Console.WriteLine("IMPORTANT TIP!: Make ABSOLUTLEY CERTAIN that the video is flowing to the Preview URL before continuing!");
+    Console.WriteLine("Press enter to continue...");
+
+    Console.Out.Flush();
+    var ignoredInput = Console.ReadLine();
+
+
+    MediaAssetFilterData drvAssetFilter = new MediaAssetFilterData
+    {
+        PresentationTimeRange = new PresentationTimeRange
+        {
+            ForceEndTimestamp = false,
+            // 10 minute (600) seconds sliding window
+            PresentationWindowDuration = 6000000000L,
+            // This value defines the latest live position that a client can seek back to 2 seconds, must be smaller than sliding window.
+            LiveBackoffDuration = 100000000L
         }
+    };
+    await mediaServicesAccount.GetMediaAsset(assetName).Value.GetMediaAssetFilters().CreateOrUpdateAsync(
```

Comment:
Man these lines look a lot longer now than they used to.... Any formatting we can do on them to make them cleaner looking? I like tidy